cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -149,17 +147,22 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
       initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED",
         "CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'")
       p("Roles can be created and managed using a set of Cypher administration commands executed against the `system` database.")
+      p("include::role-management-syntax.asciidoc[]")
       note {
-        p("""The role name `PUBLIC` is not permitted. This is a system-reserved role, and cannot be used.""")
+        p(
+          """There exists a special built-in role, `PUBLIC`, which is assigned to all users upon creation.
+            |This role cannot be dropped or revoked from any user, but its privileges may be modified.
+            |By default, it has the privilege to <<administration-security-administration-database-access, access>> the default database.
+            |""".stripMargin)
       }
-      p("include::role-management-syntax.asciidoc[]")
       section("Listing roles", "administration-security-roles-show", "enterprise-edition") {
         p("Available roles can be seen using `SHOW ROLES`.")
         query("SHOW ROLES", assertAllNodesShown("Role", column = "role")) {
           p(
             """This is the same command as `SHOW ALL ROLES`.
               |When first starting a Neo4j DBMS there are a number of built-in roles:
               |
+              |* `PUBLIC` - can access the default database, is automatically assigned to all users
```

Comment:
I propose:

* `PUBLIC` - can access the default database. This role is special in that it is automatically assigned to all users, and it cannot be dropped or revoked. However, its privileges can be modified. By default, it is assigned the <<administration-security-administration-database-access, `ACCESS`>> privilege on the default database.

Once this is merged, the corresponding change probably needs to be made in the "Built-in roles" section of the Operations Manual (https://neo4j.com/docs/operations-manual/4.1-preview/authentication-authorization/built-in-roles/).

@davidoliverSP2 
Overall I think the default roles deserve a section on its own and not be hidden under "Listing roles" but this is part of the needed re-architecture of this chapter.

Reply:
I think it is already updated in https://github.com/neo-technology/neo4j-manual-modeling/pull/1035 :)

Will update with your proposal here :)

Reply:
The role names are case-sensitive and the PUBLIC role are named in capital letters while the others are in lowercase letters

Reply:
I understand that that's how it is :) I think it is weird.

Reply:
Well, that was the decision when PUBLIC was introduced, but yes it is slightly odd :P

Reply:
The capitalization is deliberate to highlight that this role behaves quite differently to other roles. It cannot be assigned, unassigned or removed. All other roles are only there by default, but otherwise are identical to roles created entirely by customers. They can be assigned, unassigned, removed and even recreated.

Reply:
@davidoliverSP2 Yes, we all think built-in roles needs reworking, and expecially the very long table under `SHOW PRIVILEGES`. We had ideas for this during 4.0 docs development, but did not get that far. I think for this PR we'll start by pulling out the PUBLIC role and giving it it's own section, and leave the improvements to built-in roles in general to future work. One challenge is that it is most natural to describe those in detail only after describing the privileges, but that occurs only later in the document.